tools/perf/ucc_pt_op_memcpy.cc

```diff
@@ -19,8 +20,18 @@ ucc_pt_op_memcpy::ucc_pt_op_memcpy(ucc_datatype_t dt, ucc_memory_type mt,
     has_range_     = true;
     has_bw_        = true;
 
+    if (nbufs == UCC_PT_DEFAULT_N_BUFS) {
+        nbufs = 1;
```

Comment:
maybe define "1" in a macro ?
Same for the constants used in reduce and reduce_strided

Reply:
it's used in one place and default is different for different tests, can you pls elaborate what do you want to change